packages/cdktf-cli/package.json

```diff
@@ -6,9 +6,11 @@
     "cdktf": "bundle/bin/cdktf"
   },
   "scripts": {
-    "build": "tsc",
+    "compile-build-config": "tsc build.ts",
+    "prebuild": "yarn compile-build-config",
+    "build": "tsc --noEmit",
     "postbuild": "node build.js",
-    "watch": "tsc -w",
+    "watch": "yarn compile-build-config && node build.js --watch",
     "watch-preserve-output": "tsc -w --preserveWatchOutput",
```

Comment:
This line needs to be adjusted as well, as this is invoked by the "global" watch command (i.e. when running `yarn watch` in the root of this repository).